libs/gdk/src/lib/gdk-json-input/gdk-json-input.component.html

```diff
@@ -0,0 +1,5 @@
+<textarea #jsontext matInput
+    [value]="jsonString"
+    (change)="onChange($event)" 
+    (keyup)="onChange($event)"
```

Comment:
Aren't we generating more calls to `onChange` than necessary?

Reply:
I'll put some checks to avoid this problem